skosprovider_atramhasis/providers.py

```diff
@@ -40,6 +76,8 @@ def __init__(self, metadata, **kwargs):
             * `base_url` and `scheme_id` are required.
             * `session` is optional and can be used to pass a custom
                 :class:`requests.Session`, eg. to configure caching strategies.
+            * cache_config is a optional dict. Only keys starting with
+                "skosprovider.atramhasis.cache" are relevant.
```

Comment:
I prefer not to add namespacing (skosprovider.atramhasis) here, by the time the config reaches this class we already know we're dealing with skosprovider_atramhasis and only config for the skosprovider should be passed in. Namespacing would be needed in extracting the config from eg. a pyramid ini file, so it's the job of the application instantiating the provider.

So, I would expect a cache config to look like:
```
{
 'cache.backend': 'dogpile.cache.memory'
}
```

Maybe one day we would want to have more than one caching region, so that eg. a detail (get_by_id or get_by_uri) is cached for a week (mainly used for labels I believe), while a search or expand operation is only cached for a day? So, we could have `cache.long` and `cache.short` regions here?